doc/rados/operations/erasure-code-jerasure.rst

```diff
@@ -6,7 +6,7 @@ The *jerasure* plugin is the most generic and flexible plugin, it is
 also the default for Ceph erasure coded pools. 
 
 The *jerasure* plugin encapsulates the `Jerasure
-<https://bitbucket.org/jimplank/jerasure/>`_ library. It is
+<https://github.com/tsuraan/Jerasure/>`_ library. It is
```

Comment:
@dachary i am under the impression that jerasure's upstream web is jerasure.org, but seems the domain name is expired. we failed to renew it?